### PR TITLE
Begin tidying up the deployment process.

### PR DESCRIPTION
**Pull request by hairychris**
It's better than it was but I believe we still need to work on this a bit. I think the best way of getting these packages to the offline computer is using python wheels.

Unfortunately because we are using packages with C extensions (I believe mainly for dongle support) we will either have to cross compile (extra packages for the user to install) or only support migrating to the same architecture.

Dragging the virtual environment over is probably not a good idea as the paths must match as well as the architecture matching.

### CODE REVIEW

**Comment by bargst** [comment]
Using the same path on online and offline computer is what I've done for my setup. I know it's not optimal, but it works.

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+ethereum==1.1.0
```
Comment (bargst) [inline requirements.txt]: requests is missing as a dependency

File: requirements-btchip.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+git+https://github.com/LedgerHQ/btchip-python.git@fe292218e1c9c4939071650c729770bb6b34aa1b
```
Comment (bargst) [inline requirements-btchip.txt:1]: Use tag instead of commit : git+https://github.com/LedgerHQ/btchip-python.git@v0.1.16 ?

Comment (bargst) [inline requirements-btchip.txt:1]: ecdsa module is missing

File: requirements.txt (hunk 1 above)
Comment (hairychris) [inline requirements.txt]: for some reason I always think requests is part of the standard library, adding it now.

File: requirements-btchip.txt (hunk 2 above)
Comment (hairychris) [inline requirements-btchip.txt:1]: Can't use that tag as it's not up to date, I don't think there should be any disadvantage to using a particular commit. The library looks unmaintained anyway.

Regarding the ecdsa library it should be getting installed by setup.py:
`
    extras_require = {
    'smartcard': [ 'python-pyscard>=1.6.12-4build1', 'ecdsa>=0.9' ]
    },
`

From https://github.com/LedgerHQ/btchip-python/blob/master/setup.py although perhaps I need to pass something extra to pip to install 'extras'.

**Pull request merged**